client/src/views/Home/HowToPlay.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+
+import { CloseButton, Button, LoadingIndicator } from "../../components";
```

Comment:
Modal component should be used here instead of a separate view. 

Take a look at the `ManagePlayersModal` for an example of how to create a separate component using the Modal component.